cmd/modern/root/open/ads_darwin.go

```diff
@@ -0,0 +1,14 @@
+package open
+
+import (
+	"github.com/microsoft/go-sqlcmd/cmd/modern/sqlconfig"
+	"github.com/microsoft/go-sqlcmd/internal/cmdparser"
+)
+
+type Ads struct {
```

Comment:
>Ads [](http://example.com/codeflow?start=5&length=3)

I don't think this acronym has enough public traction yet. 
At minimum it needs a go-doc-compatible comment that spells out that it's for Azure Data Studio. Renaming it AzureDataStudio doesn't seem too verbose, plus it gets the product name a bit more visibility in search.

Reply:
The name of this type follows the sub-command name, which is `sqlcmd open ads`.  In the internal\tools folder, I have renamed the type to AzureDataStudio.  I will add comments here to explain this.